docs/rst/manual/reference/forward_models.rst

```diff
@@ -1,5 +1,251 @@
 Forward models
 ==============
 
-.. todo::
-   List of available forward models (jobs)
+Default jobs
+~~~~~~~~~~~~
+
+Reservoir simulation: eclipse/flow
+..................................
+
+.. code-block:: bash
+
+    EXECUTABLE script/ecl100
+    DEFAULT <VERSION> version
+    DEFAULT <NUM_CPU> 1
+    DEFAULT OPTS ""
+    ARGLIST <ECLBASE> "--version=<VERSION>" "--num-cpu=<NUM_CPU>" OPTS
+
+There are forward models for each of the simulators eclipse100, eclipse300 and
+flow, and they are all configured the same way. The version, number of cpu and
+whether or not to ignore errors can be configured in the configuration file
+when adding the job, as such:
+
+.. code-block:: bash
+
+    FORWARD_MODEL ECLIPSE100(<ECLBASE>, <VERSION>=xxxx, <OPTS>="--ignore-errors")
+
+The :code:`OPTS` argument is optional and can be removed, thus running eclipse
+without ignoring errors
+
+Reservoir modelling: RMS
+........................
```

Comment:
I'm not entirely sure how our users actually run the RMS job. As far as I understand the usage below will work if the specify `SIMULATION_JOB RMS ...`, however, if they run it as `FORWARD_MODEL RMS ...` You have to specify it as in 
https://github.com/equinor/ert/pull/551/files#diff-c7a60eb76fd629bfc593ed837ab9f271R23-R25.
Should we also provide an example like this, to be consistent in with the documentation?

Reply:
This is too specific to the `SIMULATION_JOB` at the moment. Have a look at the ECLIPSE documentation above and try to conform to that format, while keeping the below text...